### DISCUSSION TITLE
A brainstorming about using Wiki

### DISCUSSION THREAD

**Discussion by BECATRUE** [snu-quiqcl/qiwis#64] [Ideas]
We need to decide how to use Wiki and what to write README.md and Wiki, respectively.

How about this?
- In README.md, just write 'how to use `swift` for users'.
- In Wiki, write the others (e.g. structure, toy model)

I referred to my SWPP course: [github](https://github.com/swsnu/swppfall2022-team14).

**Comment by kangz12345** [comment]
Using Wiki sounds good, but for API, we can use a automatic documentation tool.
Therefore, the documentation would break into 3 parts:
1. README: brief guide to install and using
2. Wiki: detailed structure of `swift` and examples (toy model)
3. API: maybe a HTML file generated by automatic documentation tool

**Reply by kangz12345** [reply]
> 3. API: maybe a HTML file generated by automatic documentation tool

This will be discussed more in #69.
